### ISSUE TITLE
GH_BSN_BUILD_CONTEXT not working

### ISSUE THREAD

**Issue by oreporan** [ydemetriades/cfstep-github-buildstatus#3]
I have a yml with this step:

```yml
GH_Update_BuildStatus:
    type: ydemetriades/github-buildstatus
    title: 'Creating Live PR Link'
    stage: 'upload'
    when:
      steps:
        - name: publishPr
          on:
            - success
    arguments:
      GH_BSN_BUILD_CONTEXT: 'https://myurl.com/live-pr-${{CF_PULL_REQUEST_NUMBER}}'
      GH_BSN_REPO_AUTH_USER: '${{GH_BSN_REPO_AUTH_USER}}'
      GH_BSN_REPO_AUTH_TOKEN: '${{GH_BSN_REPO_AUTH_TOKEN}}'
      CF_BUILD_STATUS: 'success'
```
However the outcome is:

<img width="745" alt="Screen Shot 2020-01-26 at 14 35 25" src="https://user-images.githubusercontent.com/2146999/73135985-1ce85580-4049-11ea-9002-6f794d068b9c.png">

This is the logs from the build step:

```
Sending request to:                                                                                              
https://api.github.com/repos/xxx/xxx/statuses/975c39fe4462fbba61bf1c73695c6ad5860d68d2
with body                                                                                                        
{'state': 'success', 'target_url': 'https://g.codefresh.io//build/5e2d96245c437f3a3644885e', 'description': 'Buil
d [5e2d96245c437f3a3644885e]', 'context': 'codefresh-ci'}   
```

The `GH_BSN_BUILD_CONTEXT` is still the default one.
I've looked at your code and everything seems ok, any ideas why it doesn't work?

**Comment by oreporan** [comment]
Found the bug, it was missing in `step.yml`

Fixed in #1 along with my feature request

**Comment by ydemetriades** [comment]
Great work @oreporan !! 😋
